src/internet/internetsearchmodel.cpp

```diff
@@ -95,7 +95,7 @@ QStandardItem *InternetSearchModel::BuildContainers(const Song &s, QStandardItem
       }
       else {
         display_text = CollectionModel::TextOrUnknown(s.effective_albumartist());
-        sort_text = CollectionModel::SortTextForArtist(s.effective_albumartist());
+        sort_text = CollectionModel::SortTextForArtist(s.effective_albumartist(), /*skip_articles=*/true);
```

Comment:
Please remove the comments

Reply:
Done (the reason for having them was https://abseil.io/tips/94).